src/docs/sdk/sample-format.mdx

```diff
@@ -0,0 +1,341 @@
+: *Object, optional.* Contains information about the runtime. Usually only used
+for platforms allowing several runtimes.
+
+It can have the following attributes:
+- `name`
+- `version`
+
+```json
+{
+  "runtime": {
+    "name": "CPython",
+    "version": "3.11.2"
+  }
+}
+```
+
+`timestamp`
+
+: *String, recommended.* The timestamp when the profile was captured by the SDK
+as [RFC 3339](https://tools.ietf.org/html/rfc3339) format. This is meant to be
+the anchor for all relative timestamps captured for the profile.
+
+Ideally, this will be the same as the field `start_timestamp` on the transaction
+so all relative timestamps for transaction and profile can be both from the same
+reference point and we don't need to correct relative timestamps.
+
+If for some reasons, the profiler implementation doesn't allow that, set it to
+the closest time to the profiler start (or to whatever the profiler gives you as
+a profile start timestamp) and it will be used to align with the transaction
+later on.
+
+`transaction`
+
+: *Object, required.* Contains information about the transaction it is
+associated with.
+
+It can have the following attributes:
+- `id`: *String, required.* Contains the transaction ID.
+- `name`: *String, required.* Contains the name of the transaction.
+- `trace_id`: *String, required.* Contains the trace ID the transaction is part
+  of.
+- `active_thread_id`: *String, required.* Contains the thread ID the transaction
+  was started on as `uint64` converted to a string. On mobile, this is usually
+  the main thread.
+
+`version`
+
+: *String, required.* Represents the version of the Sample format.
+
+Acceptable values are:
+- `"1"`
+
+### Profile data
+
+The profile data consists of a list of samples referring to stacks which in
+terms refers to frames as well as some metadata about threads. This is to keep the amount of data we transfer to the
+server as limited as possible.
+
+```json
+{
+  "profile": {
+    "samples": [
+      { ... },
+      ...
+    ],
+    "stacks": [
+      [ ... ],
+      ...
+    ],
+    "frames": [
+      { ... },
+      ...
+    ],
+    "thread_metadata": {
+      "259": {
+        ...
+      }
+    }
+  }
+}
+```
+
+When collecting a stack trace, you will:
+1. Capture the stack trace.
+2. Capture a timestamp relative to the start of the profile (set in the
+`timestamp` field).
+3. Capture the thread ID where this stack trace was captured.
+4. Create a list of indices for each frames (add it to the frame list if needed
+or use the existing frame's index).
+5. Check if that stack exists in the list of stacks and if it doesn't, add it to the list of stacks
+6. Create a sample containing the thread ID, the relative timestamp and the
+stack ID.
+
+`samples`
+
+: *List, required* Contains a list of sample object captured during execution.
+Each sample can contain the following attributes:
+
+- `elapsed_since_start_ns`: *String, required.* Contains the relative timestamp in nanoseconds since the value contained
```

Comment:
This should mention that it should be a `uint64` formatted as a string. `float` is not accepted.